ports/servoshell/minibrowser.rs

```diff
@@ -217,6 +220,23 @@ impl Minibrowser {
                 return;
             };
             let mut embedder_events = vec![];
+
+            if self.hovered_link.is_some() {
```

Comment:
```suggestion
            if let Some(status_text) = &self.status_text {
```

and then do `ui.label(status_text.clone());`